client/src/styles/chat.css

```diff
@@ -52,6 +52,10 @@
         @apply bg-red p-3 break-all will-change-auto flex gap-6 items-center;
     }
 
+    .content--reply * {
+        @apply m-0 p-0;
+    }
+
```

Comment:
please keep this to the specific files rather than use apply

Reply:
It's due to the structure. It's almost impossible to add custom classes (I.e tailwind classes) to the markdown rendered content. The easiest way was to add classes to the parent container.

Reply:
Remove the apply